modules/@angular/upgrade/test/upgrade_aot_spec.ts

```diff
@@ -994,7 +997,7 @@ export function main() {
       // create the ng1 module that will import an ng2 service
       const ng1Module = angular.module('ng1Module', [])
         .factory('ng2Service', ng2ProviderFactory(Ng2Service))
-        .factory('ng1Service', () => 'ng1 service value');
+        .value('ng1Service', 'ng1 service value');
```

Comment:
I think we were explicitly testing that factories work but I guess that `.value()` is just a proxy for creating a service anyway.